shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java

```diff
@@ -311,6 +327,31 @@ public ASTNode visitSetAutoCommit(final SetAutoCommitContext ctx) {
         return result;
     }
     
+    @Override
+    public ASTNode visitAutoCommitValue(final AutoCommitValueContext ctx) {
```

Comment:
Hi `AutoCommitSegment` is necessary for `SetAutoCommitStatement`

Reply:
done.